__tests__/localStorage/StorageUtility.spec.ts

```diff
@@ -46,22 +47,22 @@ describe("StorageUtility", () => {
       expect(value).toBe("cool");
     });
 
-    it("returns null if the item is not in storage", async () => {
+    it("returns undefined if the item is not in storage", async () => {
       const storageMock = defaultMocks.storage;
-      storageMock.get.mockReturnValueOnce(Promise.resolve(null));
+      storageMock.get.mockReturnValueOnce(Promise.resolve(undefined));
       const storageUtility = getStorageUtility({ storage: storageMock });
       const value = await storageUtility.get("key");
       expect(storageMock.get).toHaveBeenCalledWith("key");
-      expect(value).toBeNull();
+      expect(value).toBeUndefined();
     });
 
     it("throws an error if the item is not in storage and errorOnNull is true", async () => {
       const storageMock = defaultMocks.storage;
-      storageMock.get.mockReturnValueOnce(Promise.resolve(null));
+      storageMock.get.mockReturnValueOnce(Promise.resolve(undefined));
       const storageUtility = getStorageUtility({ storage: storageMock });
-      await expect(storageUtility.get("key", true)).rejects.toThrowError(
-        "key is not stored"
-      );
+      await expect(
+        storageUtility.get("key", { errorIfNull: true })
+      ).rejects.toThrowError("[key] is not stored");
```

Comment:
Very minor thing, but generally in a test I **_tend_** not to assert on the text of an error message, but I would assert that the message contains the value I expect to be in error. In other words I'd just change this to:
`).rejects.toThrowError("[key]");`
I say 'tend' above because sometimes I do want to test that it's the correct message too (which is why I like to have error messages in RDF vocabs, 'cos then every message gets a globally unique ID (i.e. it's IRI), and then I can assert on that ID), but in this case I'd probably use:
`).rejects.toThrowError("[key]", "not stored");`

Reply:
According to the documentation (https://jestjs.io/docs/en/expect.html#tothrowerror), `toThrowError` cannot take multiple arguments: only one expected substring may be provided.

Reply:
It actually came up when converting `LitVocabTerm` to TS: I had to update the tests because when plain JS ignores unexpected parameters, TS complains about them :)

Reply:
Ah Ok - but you know what I mean :) ! (I assume there is an easy Jest way to assert that the text of an Error contains multiple strings (ChaiJS has it, which is probably where I used it before: https://github.com/chaijs/chai/issues/858)...?)